### PR TITLE
add log and verbose in command line argument, save images only in log…

### PR DESCRIPTION
**Pull request by brichet**
… mode

### CODE REVIEW

**Comment by artzet-s** [comment]
Hi, Nicolas, 

Can you update the test (./test/test_eartrack.py) following your modification ? The test correspond to the notebook execution without graphical mode.

**Comment by pradal** [comment]
Nicolas (@brichet), please modify the test and notebook by adding the log as input parameter.
Now, there is a 4th parameter (log) without default value.

File: src/openalea/eartrack/binarisation_folder.py [hunk 1]
```diff
@@ -10,8 +10,7 @@
 import openalea.eartrack.binarisation as bin
 
 
-def init(param_folder, input_folder, output_folder):
-    # images_folder = os.path.join(input_folder, 'images')
+def init(param_folder, input_folder, output_folder, log):
```
Comment (pradal) [inline src/openalea/eartrack/binarisation_folder.py]: I will set log to False by default:
def init(param_folder, input_folder, output_folder, log=False):

Comment (brichet) [reply src/openalea/eartrack/binarisation_folder.py]: OK

**Pull request merged**